koji_containerbuild/cli.py

```diff
@@ -149,6 +151,9 @@ def parse_arguments(options, args, flatpak):
     if build_opts.signing_intent and build_opts.compose_ids:
         parser.error(_("--signing-intent cannot be used with --compose-id"))
 
+    if build_opts.compose_ids and build_opts.yum_repourls:
+        parser.error(_("--compose-id cannot be used with --repo-url"))
```

Comment:
This line doesn't get test coverage (actually neither does line 152).

Reply:
Added a unit test.